src/components/pagination/WPagination.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <nav class="flex items-center gap-xs" aria-label="Pagination">
+    <button
+      type="button"
+      :disabled="data.isFirstPage"
+      @click="data.prev"
+      :class="{
+        'opacity-50': data.isFirstPage,
+        'pointer-events-none': data.isFirstPage
+      }"
+      class="min-h-[38px] min-w-[38px] py-sm px-sm inline-flex justify-center items-center gap-sm text-sm rounded-lg hover:bg-wit-blue-400 focus:outline-none"
+      aria-label="Previous"
+    >
+      <svg
+        class="shrink-0 size-3.5"
+        xmlns="http://www.w3.org/2000/svg"
+        width="24"
+        height="24"
+        viewBox="0 0 24 24"
+        fill="none"
+        stroke="currentColor"
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      >
+        <path d="m15 18-6-6 6-6"></path>
+      </svg>
+      <span>Previous</span>
+    </button>
+    <div class="flex items-center gap-x-xs">
+      <button
+        v-for="p in data.pageCount"
+        :key="p"
+        type="button"
+        class="min-h-[38px] min-w-[38px] flex justify-center items-center hover:bg-wit-blue-400 py-sm px-sm text-sm rounded-lg"
+        :class="{
+          'outline-none': p === page,
+          'bg-wit-blue-500': p === page,
+          'pointer-events-none': p === page
+        }"
+      >
+        {{ p }}
+      </button>
+    </div>
+    <button
+      type="button"
+      :disabled="data.isLastPage"
```

Comment:
Pagination does not correctly work in storybook, maybe because of this this error:
```Property 'isLastPage' does not exist on type '{ currentPage: number; currentPageSize: number; pageCount: number; isFirstPage: boolean; prev: () => void; next: () => void; }'.```

Reply:
I think it's a type issue because `isLastPage` works and it's in the [documentation](https://vueuse.org/core/useOffsetPagination/#useoffsetpagination). I still have to find a solution for this

Reply:
The problem was that it was infering the `UseOffsetPaginationInfinityPageReturn` instead of `UseOffsetPaginationReturn`